packages/page-nfts/src/AccountItems/Item.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2022 @polkadot/app-assets authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ItemInfo } from '@polkadot/app-nfts/AccountItems/types';
+
+import React from 'react';
+
+import { AddressSmall, IconLink } from '@polkadot/react-components';
+import { formatNumber } from '@polkadot/util';
+
+interface Props {
+  className?: string;
+  collectionName: string;
+  value: ItemInfo;
+}
+
+function Item ({ className, collectionName, value: { account, id, ipfsData } }: Props): React.ReactElement<Props> {
+  const name = ipfsData?.name || collectionName;
+  const imageLink = ipfsData?.image ? `https://ipfs.io/ipfs/${ipfsData.image}` : '';
```

Comment:
That gateway is extremely slow, if we decide to display the images on that page - it would be better to create a simple proxy/CDN service to cache them. And yes, we would need to make that gateway's URL configurable